src/silverlabnwb/metadata.py

```diff
@@ -42,6 +42,13 @@ def read_user_config():
     return settings, comments
 
 
+def read_custom_config(config_file):
+    """Read the metadata from a user-provided YAML file."""
+    with open(config_file) as config:
+        settings, comments = read_base_config_file(config)
+    return settings
```

Comment:
should this be `settings, _ = read_base_config_file(config)` to be clear that we shouldn't use the comments here.